installer/lib/kitto_new.ex

```diff
@@ -70,7 +71,32 @@ defmodule Mix.Tasks.Kitto.New do
   end
 
   @moduledoc """
-  Kitto installer. Generates a scaffolding to build a new dashboard upon.
+  Creates a new Kitto dashboard.
+
+  It expects the path of the project as argument.
+
+      mix kitto.new PATH [--edge] [--dev KITTO_PATH]
```

Comment:
I disagree. The argument is the path for the application. The name is assumed based on the path.